xc7/primitives/common_slice/common_slice.pb_type.xml

```diff
@@ -297,7 +297,7 @@
     </mux>
 
     <!-- [A-D]MUX -->
-    <mux name="DMUX"
+    <mux name="DOUTMUX"
```

Comment:
This change is fine, but doesn't really fit in this PR at all?

Reply:
Done - Moved to https://github.com/SymbiFlow/symbiflow-arch-defs/pull/726